.craft.yml

```diff
@@ -34,6 +34,12 @@ targets:
         if OS.mac?
           url "https://downloads.sentry-cdn.com/sentry-cli/{{version}}/sentry-cli-Darwin-universal"
           sha256 "{{checksums.sentry-cli-Darwin-universal}}"
+        elsif Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
+          url "https://downloads.sentry-cdn.com/sentry-cli/{{version}}/sentry-cli-Linux-aarch64"
+          sha256 "{{checksums.sentry-cli-Linux-aarch64}}"
+        elsif Hardware::CPU.arm?
+          url "https://downloads.sentry-cdn.com/sentry-cli/{{version}}/sentry-cli-Linux-armv7"
+          sha256 "{{checksums.sentry-cli-Linux-armv7}}"
```

Comment:
How would you feel if we start nesting this to show the structure more clearly? 

- if mac? -> universal
- elsif linux?
  - if arm?
    - if 64bit -> aarch64
    - else -> armv7
  - elsif intel?
    - if 64bit? -> linux x64
    - else -> linux x86
  - else -> unsupported arch
- else -> unsupported OS